src/test/java/seedu/address/model/employee/LeaveTest.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.employee;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class LeaveTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Leave(null));
+    }
+
+    @Test
+    public void constructor_invalidPhone_throwsIllegalArgumentException() {
```

Comment:
Test naming can be improved to be constructor_invalidLeave_throwsIllegalArgumentException()